### PR TITLE
Remove Grunt

### PR DESCRIPTION
**Pull request by nathunsmitty**
## Types of changes

- [ ] Bugfix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [x] Refactor (change which changes the codebase without affecting its external behavior)
- [x] Non-breaking change (fix or feature that would causes existing functionality to work as expected)
- [ ] Breaking change (fix or feature that would cause existing functionality to __not__ work as expected)

## Purpose

Removes our dependency on Grunt, relying instead on just Webpack.

## Checklist

- [ ] My branch follows the branch naming scheme of UCLA Radio, and can merge into `master` without error.
- [ ] My code follows the code style of this project, and I have linted it to confirm this.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,10 @@
   "scripts": {
     "postcheckout": "yarn",
     "start": "node ./bin/www",
-    "dev": " concurrently --kill-others \"mongod\" \"grunt watch\" \"nodemon ./bin/www\" ",
+    "watch": "nodemon ./bin/www --ignore public/",
+    "dev":
```
Comment (binerys) [inline package.json]: Should `mongod` still be added here? Or is it better practice to have the developer run mongod separately? Also how about the `--kill-others` command?

Comment (nathunsmitty) [reply package.json]: I haven't really found any other projects that start mongo in package.json and it does clutter the output quite a bit so I was thinking we could cut it.

Comment (nathunsmitty) [reply package.json]: We could add `--kill-others` back though!

**Comment by ChrisLaganiere** [comment]
Fare you well, grunt 🙂

**Pull request merged**